### PR TITLE
Fix/remove invalid HTML markup in Rails views

### PR DESCRIPTION
**Pull request by myabc**

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://myabc-fix-invalid-html-markup.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://myabc-fix-invalid-html-markup.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: app/views/wiki_menu_items/edit.html.erb [hunk 1]
```diff
@@ -57,7 +57,7 @@ See doc/COPYRIGHT.rdoc for more details.
       <%= form.check_box 'new_wiki_page' %>
       <%= form.label 'new_wiki_page', l(:label_wiki_show_new_page_link) %>
 
-      </br>
+      <br>
```
Comment (NobodysNightmare) [inline app/views/wiki_menu_items/edit.html.erb:60]: Earlier in my life, when I stopped looking at HTML, the new thing was that self-closing tags should be apparent. Done like this: `<br />` (I think what the original author intended to do).

At the time, `<br>` was the old thing, that was deprecated.

What is the current way of doing things? I thought `<br />` was still hot...

Comment (myabc) [inline app/views/wiki_menu_items/edit.html.erb:60]: @NobodysNightmare HTML 5 doesn't require XHTML syntax. I think it's safe to move away from this syntax, although there are some gotchas when it comes to Angular tag directives.

http://www.techrepublic.com/blog/10-things/10-things-you-should-know-about-html5/

**Comment by myabc** [comment]
> Earlier in my life

You're still young!

**Pull request merged**